packages/coordinator/ts/common/networks.ts

```diff
@@ -1,64 +1,84 @@
+import {
```

Comment:
just thinking about the frontend interface would also need these information, maybe put this to a place that both interface and coordinator can call? (no need to do in this PR)

Reply:
good point, maybe maci-cli/sdk?